packages/shared/lib/services/sync/config/deploy.service.ts

```diff
@@ -268,33 +269,41 @@ export async function upgradePreBuilt({
     delete flowData.id;
 
     try {
-        const [syncId] = await db.knex.from<SyncConfig>(TABLE).insert(flowData).returning('id');
```

Comment:
change below is not entirely related but I fix the naming: the query returns SyncConfigs not Sync. I also added a the `newSyncConfigId` to avoid casting